backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -16,28 +20,37 @@
 @Service
 @Transactional
 public class ApiUserService {
+    @Autowired
+    private SimpleReportProperties _props;
+    private ApiUserRepository _apiUserRepo;
+    private IdentitySupplier _supplier;
 
-	private ApiUserRepository _apiUserRepo;
-	private IdentitySupplier _supplier;
+    private static final Logger LOG = LoggerFactory.getLogger(ApiUserService.class);
+    
+    public ApiUserService(ApiUserRepository apiUserRepo, IdentitySupplier supplier) {
+        _apiUserRepo = apiUserRepo;
+        _supplier = supplier;
+    }
 
-	private static final Logger LOG = LoggerFactory.getLogger(ApiUserService.class);
-	
-	public ApiUserService(ApiUserRepository apiUserRepo, IdentitySupplier supplier) {
-		_apiUserRepo = apiUserRepo;
-		_supplier = supplier;
-	}
+    public boolean isAdminUser() {
+        IdentityAttributes userIdentity = _supplier.get();
+        if (!_props.getAdminEmails().contains(userIdentity.getUsername())) {
+            throw new IllegalGraphqlArgumentException("No active test order was found for that patient");
```

Comment:
It seems like we can't decide if we're checking it or asserting it—asserting it is probably better?